monkCLI/src/commands/exportContainerCommand.php

```diff
@@ -71,6 +71,8 @@ private function downloadMedia($client, $containerName, $savePath, OutputInterfa
         $progress = new ProgressBar($output, $numberOfFiles);
         $progress->start();
         ini_set('memory_limit', -1);
+
+        $filesDownloaded = 0;
```

Comment:
being picky, but could you move this to be right below `$numberOfFiles` without the extra line break :)